pkg/session/bootstrap.go

```diff
@@ -611,8 +612,9 @@ const (
 		step INT(11),
 		target_scope VARCHAR(256) DEFAULT "",
 		error BLOB,
+		modify_params json,
```

Comment:
Is this column necessary for history table?

Reply:
just make sure they have the same table structure, we have a UT for this, and when transfering to history, we use `insert into ... select *`